mscore/musescore.cpp

```diff
@@ -2560,6 +2560,7 @@ void MuseScore::setCurrentScoreView(ScoreView* view)
 
       QAction* a = getAction("concert-pitch");
       a->setChecked(cs->styleB(Sid::concertPitch));
+      a->setText(a->isChecked() ? "Transposed" : "Concert Pitch");
```

Comment:
I think it needs `tr()` here too: `a->setText(a->isChecked() ? tr("Transposed") : tr("Concert Pitch"));`

Reply:
Ah yeah I forgot that one